dags/process_elife_zip_dag.py

```diff
@@ -261,7 +262,13 @@ def send_article_to_content_service(**context) -> None:
         data=etree.tostring(libero_xml, xml_declaration=True, encoding='UTF-8'),
         headers={'content-type': 'application/xml'}
     )
-    response.raise_for_status()
+
+    try:
+        response.raise_for_status()
+    except HTTPError as e:
+        logger.error('%s HTTP Error= %s', response.status_code, response.text)
```

Comment:
could include something about the request here (e..g the URL) to help correlate

Reply:
yep that information will be logged when re-raising the error.